addons/mail/static/src/models/composer/composer.js

```diff
@@ -289,82 +284,120 @@ function factory(dependencies) {
             body = this._generateMentionsLinks(body);
             body = parseAndTransform(body, addLink);
             body = this._generateEmojisOnHtml(body);
+            return body;
+        }
+
+        /**
+         * Recursive function that process the queue.
+         */
+        async processMessageQueue() {
```

Comment:
probably to move on thread

Reply:
The reason the queue (renamed `pendingMessagesToBeSent`) is on thread is because we need to display the temporary message.
So we could move everything related to posting message inside the thread model. It's related to the composer for me but could make sens on thread. 
Also, maybe we don't want to move to much stuff around in stable ?